### PR TITLE
Update pytorch example's readme

### PR DESCRIPTION
**Pull request by qiuxin2012**

### CODE REVIEW

File: apps/pytorch/README.md [hunk 1]
```diff
@@ -2,19 +2,21 @@
...
-* Analytics-Zoo 0.6.0-SNAPSHOT.dev4 and above
-* Jupyter Notebook
-* Ray, psutil(Will be removed in the future)
+* Analytics-Zoo 0.6.0-SNAPSHOT.dev5 and above
+* Jupyter Notebook, matplotlib
 
-## Install Analytics Zoo
-We recommend you to use Anaconda env to prepare the enviroments. Especially, you want to run on a yarn cluster(yarn-client mode). 
+## Prepare environments
+We recommend you to use [Anaconda](https://www.anaconda.com/distribution/#linux) to prepare the enviroments. Especially, you want to run on a yarn cluster(yarn-client mode).
```
Comment (jason-dai) [inline apps/pytorch/README.md]: We recommend you to use ..., especially if you want to run on a yarn cluster (yarn-client mode)

File: apps/pytorch/README.md [hunk 2]
```diff
@@ -2,19 +2,21 @@
 
 This example is migrated from [PROGRESSIVE GROWING OF GANS](https://pytorch.org/hub/facebookresearch_pytorch-gan-zoo_pgan/), and demonstrates how to run distributed inference using a pre-trained Pytorch Model.
 
-## Environment
+## Requisition
```
Comment (jason-dai) [inline apps/pytorch/README.md]: Requirement

File: apps/pytorch/README.md [hunk 3]
```diff
@@ -23,9 +25,9 @@ If you want to run spark local, just start jupyter notebook:
 jupyter notebook
```
Comment (cyita) [inline apps/pytorch/README.md]: In local mode, the Run Generation in Distributed Mode section gives an error.
Env: Analytics-Zoo 0.6.0.dev5
Error msg:
Py4JJavaError: An error occurred while calling z:org.apache.spark.api.python.PythonRDD.collectAndServe.
: org.apache.spark.SparkException: Job aborted due to stage failure: Task 1 in stage 0.0 failed 1 times, most recent failure: Lost task 1.0 in stage 0.0 (TID 1, localhost, executor driver): java.io.IOException: Cannot run program "/root/anaconda2/envs/facegen/bin/python
": error=2, No such file or directory

**Comment by qiuxin2012** [comment]
Manual test passed.

**Pull request merged**